schema/rdl_schema_0.1.json

```
@@ -1234,9 +1263,11 @@
             "type": "number"
           },
           "minItems": 2,
-          "maxItems": 2
+          "maxItems": 2,
+          "uniqueItems": true
```

Coordinates can be non-unique:

```suggestion
          "maxItems": 2
```